NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# Cf my products

# Description

Added a my products page under user profile. You should be able to CRD all of your products. 


## Related Ticket(s)
19, 15(kind of)

## Steps to Test
Merge and commit on your branch. Checkout cfMyProducts, then run your server. Login and navigate to my products, then create a product. Delete that product. Check that all other features work.


# Checklist:
